openbas-front/src/admin/components/scenarios/scenario/ScenarioContext.ts

```diff
@@ -39,6 +39,11 @@ const injectContextForScenario = (scenario: ScenarioStore) => {
         resolve(response.data);
```

Comment:
I didn't see that before. Here, I'd prefer to use **await** rather than creating another promise within a then block (l34-l39)